frontend/src/app/testResults/uploads/Uploads.tsx

```diff
@@ -210,7 +246,7 @@ const Uploads = () => {
               </div>
             </div>
           )}
-          {errors && errors.length > 0 && (
+          {fileInputResetValue > 0 && !reportId && (
```

Comment:
this allows us to see "File not accepted" and whatever is in `errorMessageText` in cases where there's an error but `errors` is empty (e.g. server upload failure)

Reply:
can we use the `errorMessageText` to decide to show this? initialized it to null, and reset it back to null upon submitting. seems kinda weird it has been initialized to an error message

Reply:
yeah I like that better - I don't think there's a case where the initialized message even shows